Release Cognitive Services Data Plane SDKs Round I (#4955)
Azure/azure-sdk-for-js: sdk/cognitiveservices/cognitiveservices-contentmoderator/README.md

#### nodejs - Authentication, client creation and getDetails listManagementImageLists as an example written in TypeScript.

##### Install @azure/ms-rest-azure-js

```bash
npm install @azure/ms-rest-azure-js
```

##### Sample code

> The following sample lets you detect the langauge of text provided using text moderator APIs. To know more, Refer the [Azure Documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/content-moderator/overview)

Reviewer: ```suggestion
The following sample detects the langauge of text provided using text moderator APIs. To know more, refer to the [Azure Documentation on Content Moderator](https://docs.microsoft.com/en-us/azure/cognitive-services/content-moderator/overview)
```
